Add the capability to know which CVE’s affect an agent - Documentation changes (#3564)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/vuln-detector.rst

| **Allowed values**   | yes, no   |
+----------------------+-----------+

.. _vuln_det_ignore_time:

ignore_time
^^^^^^^^^^^^

Time during which vulnerabilities that have already been alerted will be ignored. When this time hasn't passed yet, only :ref:`partial scans <vuln_det_scan_types>` will be performed.

+----------------------+------------------------------------------------------------------------------------------------------------------------------------+
| **Default value**    | 6 hours                                                                                                                            |
+----------------------+------------------------------------------------------------------------------------------------------------------------------------+
| **Allowed values**   | A positive number that should contain a suffix character indicating a time unit: s (seconds), m (minutes), h (hours) or d (days).  |
+----------------------+------------------------------------------------------------------------------------------------------------------------------------+

> .. note:: In a partial scan new packages are scanned, but only a full scan removes the CVEs related to old packages that aren't present any more.

Reviewer: ```suggestion
.. note:: In a partial scan new packages are scanned, but only a full scan removes the CVEs related to old packages that are not present anymore.
```
